client/components/foldable-card/index.jsx

```diff
@@ -120,12 +120,16 @@ class FoldableCard extends Component {
 			'is-clickable': !! this.props.clickableHeader,
 			'has-border': !! this.props.summary,
 		} );
+		const header = createElement(
+			this.props.headerTag ?? 'span',
```

Comment:
Super nit:

1. I checked the codebase, it seems that we're using the term `tagName` for this kind of props, maybe let's also follow it and call it `headerTagName`?
2. Add `headerTag` to the `propTypes` above?
3. Move `?? span` to the `defaultProps` above? Not sure if this is the correct convention for Calypso, cc: @roo2

Reply:
You're right, it should be added to `propTypes`!